packages/node_modules/@webex/react-component-adaptive-card/src/adaptiveCard.scss

```diff
@@ -150,6 +150,31 @@
   }
 
   .ac-textInput {
+    .md-input__wrapper{
+    box-sizing: border-box;
+    display: block;
+    flex-grow: 1;
+    position: relative;
+    padding: 0;
+    .md-input__after, .md-input__before{
+    -webkit-box-sizing: inherit;
+    -moz-box-sizing: inherit;
+    box-sizing: inherit;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+    flex-flow: row nowrap;
+    position: absolute;
+    height: 100%;
+    top: 0;
+    width: 2.75rem
+    }
+    .md-input__after{
+    right: 0;
+    left: auto;
+    }
+  }
```

Comment:
Can you please adjust the indentations of these? The alignment is not correct.

Reply:
@adamweeks added indentations